build.gradle

```diff
@@ -218,19 +214,15 @@ task copyToLibs(type: Copy) {
 }
 
 task copyToRoot(type: Copy, dependsOn: [copyToLibs, jar, converterJar]) {
-	outputs.files.setFrom(file("$buildDir/libs/pcgen-${version}.jar"))
```

Comment:
This change causes the copyToRoot task to fail in gradle 2.2 and 2.10. Again it is one of those odd workarounds I needed to do.